bmq-sdk/src/test/java/com/bloomberg/bmq/it/SessionIT.java

```diff
@@ -3590,7 +3591,7 @@ public void testQueueFlushDown() throws Exception {
             // 4) Pack a number of PUT messages
             PutMessage[] putMessages = new PutMessage[NUM_MESSAGES];
             for (int i = 0; i < putMessages.length; i++) {
-                byte[] payload = new byte[4096];
```

Comment:
4096x500 ~= 2MB does not fit in the default byte quota for `bmq.test.mmap.priority`
Previously message compression allowed that